### PR TITLE
Documented using Skaffold with RD

### PR DESCRIPTION
**Pull request by jtravee**
Reference: [New How-to Guide](https://docs.rancherdesktop.io/how-to-guides/skaffold-and-rancher-desktop)

### CODE REVIEW

File: docs/how-to-guides/skaffold-and-rancher-desktop.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Skaffold and Rancher Desktop
+---
+
+Skaffold is a command line tool that optimizes your development workflow by automating the creation, configuration, and debugging of your Kubernetes cluster. Skaffold also provides an efficient way of building, pushing, testing, and deploying your app.
```
Comment (gunamata) [inline docs/how-to-guides/skaffold-and-rancher-desktop.md:5]: I would just use the introduction from Skaffold docs as below... 


Skaffold is a command line tool that facilitates continuous development for Kubernetes-native applications. Skaffold handles the workflow for building, pushing, and deploying your application, and provides building blocks for creating CI/CD pipelines. This enables you to focus on iterating on your application locally while Skaffold continuously deploys to your local or remote Kubernetes cluster. To know more about skaffold, refer to the project docs [here](https://skaffold.dev/docs/)

File: docs/how-to-guides/skaffold-and-rancher-desktop.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+1. Run `skaffold init`.
+
+    Per the [Skaffold docs](https://skaffold.dev/docs/pipeline-stages/init/#build-config-initialization),`skaffold init` walks through your project directory and looks for any build configuration files such as `Dockerfile`, `build.gradle/pom.xml`, `package.json`, `requirements.txt`, or `go.mod`. 
+    
+    We will select `Dockerfile` and `package.json` in our example. This will generate the initial configuration file that you can modify as needed. When prompted, select `yes` to write your config to `skaffold.yaml`. 
+
+1. In your editor, review your `app.js` and `manifests.yaml` files. Note that in `manifests.yaml`, you will have a deployment config as well as a service config. It is only necessary to have 1 `replica` for testing purposes.
+
+1. Back in your terminal, you'll notice that you will have two options: `skaffold run` that lets you build and deploy, and `skaffold dev` that allows you to enter development mode with auto-redeploy. Choose `skaffold dev` for this example.
```
Comment (gunamata) [inline docs/how-to-guides/skaffold-and-rancher-desktop.md:28]: I was wrong about skipping the note to connect to registry before running `skaffold dev`.. There was actually someone asking about this in [RD discussions](https://github.com/rancher-sandbox/rancher-desktop/discussions/1783) yesterday.. That changed my mind, so we can add a note on below lines..

As you need to have push access to the image repository, you can either docker login (if you have docker hub account) before running `skaffold dev` or you can setup a local registry using `docker run -d -p 5000:5000 --restart=always --name registry registry:2` and then run `skaffold dev --default-repo=localhost:5000`.

File: docs/how-to-guides/skaffold-and-rancher-desktop.md (hunk 1 above)
Comment (jtravee) [reply docs/how-to-guides/skaffold-and-rancher-desktop.md]: Updated as suggested.

File: docs/how-to-guides/skaffold-and-rancher-desktop.md (hunk 2 above)
Comment (jtravee) [reply docs/how-to-guides/skaffold-and-rancher-desktop.md]: Added commands as suggested (created tabs for easier reading).

File: docs/how-to-guides/skaffold-and-rancher-desktop.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    Per the [Skaffold docs](https://skaffold.dev/docs/pipeline-stages/init/#build-config-initialization),`skaffold init` walks through your project directory and looks for any build configuration files such as `Dockerfile`, `build.gradle/pom.xml`, `package.json`, `requirements.txt`, or `go.mod`. 
+    
+    We will select `Dockerfile` and `package.json` in our example. This will generate the initial configuration file that you can modify as needed. When prompted, select `yes` to write your config to `skaffold.yaml`. 
+
+1. In your editor, review your `app.js` and `manifests.yaml` files. Note that in `manifests.yaml`, you will have a deployment config as well as a service config. It is only necessary to have 1 `replica` for testing purposes.
+
+1. Back in your terminal, you'll notice that you will have two options: `skaffold run` that lets you build and deploy, and `skaffold dev` that allows you to enter development mode with auto-redeploy. We will use `skaffold dev` in this example.
+
+   As you will need to have push access to the image repository, you can use either use your docker login, or you can set up a local registry:
```
Comment (gunamata) [inline docs/how-to-guides/skaffold-and-rancher-desktop.md:31]: `...use either use..`

duplicate of the word `use` needs to be fixed.

Comment (jtravee) [reply docs/how-to-guides/skaffold-and-rancher-desktop.md]: Thank you, I removed it!

**Review by gunamata — APPROVED** [review]
👍

**Pull request merged**